tests/unit_tests/service/test_authentication.py

```diff
@@ -132,3 +129,12 @@ def test_processes_valid_token(
 ):
     inner = main.verify_access_token(oidc_config)
     inner(access_token=valid_token_with_jwt["access_token"])
+
+
+def test_session_cache_manager_returns_writable_file_path(tmp_path):
+    with patch("os.environ.get") as xdg_directory:
+        xdg_directory.return_value = tmp_path
```

Comment:
Is there a way to make this return_value only used when querying for `"XDG_CACHE_HOME"`?